GULNIHAL_NALDOKEN.md

```diff
@@ -0,0 +1,5 @@
+Gulnihal
+I generally don't do sports. I had aerobics and steps classes in the past. I like watching tennis matches.
+My favourite food is Turkish dolma. It's easy to cook and very delicious and it's also one of my best recipies that I cook.
+Here is the link for my favourite music https://www.youtube.com/watch?v=LwtPS5Ap4Lk
+I've graduated from the top university of Turkey which accepts students who get the highest marks in the university entrance exam. (It's found quite impressive by people around me :) ) 
```

Comment:
Glad to know about one of my favorite and hardworking classmate. Well done I'm impressed and you're excellent student in this class too.